subprojects/test-runner/device-provider/impl/build.gradle.kts

```diff
@@ -8,12 +8,9 @@ plugins {
     id("convention.libraries")
 }
 
-publish {
```

Comment:
Delete because I don't know for what reason this exists

Reply:
It defines artifactId of maven module, when you applying a dependency in project it's:

`com.avito.android:<WHAT IS SET HERE>:version`

Reply:
As I understand if Integration compile in Avito was done that mean everything is ok

Reply:
yes